.github/pull_request_template.md

```diff
@@ -23,10 +23,5 @@
 ### Additional Notes
 <!-- Anything else we should know when reviewing? -->
 
-### Security 
-Datadog employees:
-- [ ] If this PR touches code that signs or publishes builds or packages, or handles credentials of any kind, I've requested a review from `@DataDog/security-design-and-guidance`.
-- [ ] This PR doesn't touch any of that.
-
 Unsure? Have a question? Request a review!
```

Comment:
The "Unsure?…" line was part of the security template too, it should be removed as well.

Reply:
Done